tasks/setup-rocky-8.yml

```diff
@@ -2,22 +2,57 @@
 # Copyright (C) 2021 Robert Wimmer
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-- name: (Rocky Linux 8) Install EPEL & ELRepo repository
-  ansible.builtin.yum:
-    name:
-      - epel-release
-      - elrepo-release
-    update_cache: true
+- name: (Rocky Linux 8) Tasks for standard kernel
+  block:
+    - name: (Rocky Linux 8) Install EPEL & ELRepo repository
+      ansible.builtin.yum:
+        name:
+          - epel-release
+          - elrepo-release
+        update_cache: true
 
-- name: (Rocky Linux 8) Ensure WireGuard DKMS package is removed
-  ansible.builtin.yum:
-    name:
-      - "wireguard-dkms"
-    state: absent
+    - name: (Rocky Linux 8) Ensure WireGuard DKMS package is removed
+      ansible.builtin.yum:
+        name:
+          - "wireguard-dkms"
+        state: absent
 
-- name: (Rocky Linux 8) Install WireGuard packages
-  ansible.builtin.yum:
-    name:
-      - "kmod-wireguard"
-      - "wireguard-tools"
-    state: present
+    - name: (Rocky Linux 8) Install WireGuard packages
+      ansible.builtin.yum:
+        name:
+          - "kmod-wireguard"
+          - "wireguard-tools"
+        state: present
+  when:
+    - (wireguard_rockylinux8_installation_method == "standard" and ansible_architecture == "x86_64" and ansible_kernel is match("4\.18\.0-.*x86_64")) or
```

Comment:
That's way too specific. I don't support any other architecture then `amd64` as I'm not able to test it and therefore not able to maintain it. If there are three use cases like `standard', `kmod` and 'dkms' then there should be three `block`'s. If someone has an `arm64` kernel and wants to use one of the three methods then he/she can do so.

Reply:
Fair enough. I'm reverting back to `standard`/`dkms`as it was.  
This is good enough for me to use. For my use case, I'll find another way to dynamically setup the variable before running the role.

If everything is in order, I'm done editing my branch. Thank you!